qa/tasks/cephfs/filesystem.py

```diff
@@ -615,9 +615,14 @@ def required_client_features(self, *args, **kwargs):
         c = ["fs", "required_client_features", self.name, *args]
         return self.mon_manager.run_cluster_cmd(args=c, **kwargs)
 
-    # In Octopus+, the PG count can be omitted to use the default. We keep the
-    # hard-coded value for deployments of Mimic/Nautilus.
-    pgs_per_fs_pool = 8
+    # Since v15.1.0 the pg autoscale mode has been enabled as default,
+    # will let the pg autoscale mode to calculate the pg_num as needed.
+    # We set the pg_num_min to 64 to make sure that pg autoscale mode
+    # won't set the pg_num to low to fix Tracker#45434.
+    pg_num = 512
```

Comment:
This is too many. We end up with `PG_DEGRADED` warnings like:

https://pulpito.ceph.com/?branch=wip-pdonnell-testing-20210608.022222

I think because the OSDs become too slow with so many PGs. Why doesn't something like `64` work?

Reply:
I didn't try the 64 yet after reading that ceph doc and with whose suggestion I just set it to 512.
Let me have a try.

Reply:
There still has the following issue:

`"2021-06-10T03:01:46.635107+0000 mon.a (mon.0) 1289 : cluster [WRN] Health check failed: Reduced data availability: 1 pg peering (PG_AVAILABILITY)" in cluster log`

https://pulpito.ceph.com/xiubli-2021-06-10_02:37:32-fs:functional:-wip-lxb-20210607-51113-unknown-mtype-distro-basic-smithi/